datasets/offcombr/offcombr.py

```diff
@@ -0,0 +1,110 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""OffComBR: an annotated dataset containing for hate speech detection in Portuguese composed of news comments on the Brazilian Web."""
+
+from __future__ import absolute_import, division, print_function
+
+import datasets
+
+
+_CITATION = """\
+@article{Pelle2017,
+title={Offensive Comments in the Brazilian Web: a dataset and baseline results},
+author={Rogers P. de Pelle and Viviane P. Moreira},
+booktitle={6th Brazilian Workshop on Social Network Analysis and Mining (BraSNAM)},
+year={2017},
+}
+"""
+
+_DESCRIPTION = """\
+OffComBR: an annotated dataset containing for hate speech detection in Portuguese composed of news comments on the Brazilian Web.
+"""
+
+_HOMEPAGE = "http://www.inf.ufrgs.br/~rppelle/hatedetector/"
+
+_LICENSE = "Unknown"
+
+_URLs = {
+    "offcombr-2": "https://github.com/rogersdepelle/OffComBR/raw/master/OffComBR2.arff",
+    "offcombr-3": "https://github.com/rogersdepelle/OffComBR/raw/master/OffComBR3.arff",
+}
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class NewDataset(datasets.GeneratorBasedBuilder):
+    """OffComBR: an annotated dataset containing for hate speech detection in Portuguese composed of news comments on the Brazilian Web."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="offcombr-2",
+            version=VERSION,
+            description="OffComBR-2 contains samples with Fleiss Kappa measure between annotators of 0.71",
+        ),
+        datasets.BuilderConfig(
+            name="offcombr-3",
+            version=VERSION,
+            description="OffComBR-3 only contains instances for which the class has been agreed by all three judges",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = (
+        "offcombr-2"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+    )
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "label": datasets.Value("string"),
```

Comment:
Can you use the ClassLabel feature type for the label field please ?
For example
```
"label": datasets.ClassLabel(names=["no", "yes"])
```